.github/workflows/e2e-tf-deployment.yaml

```diff
@@ -118,7 +126,9 @@ jobs:
           EOF
 
           # Deploy infrastructure, install helm charts, run e2e tests, and cleanup all
-          aws configure set aws_access_key_id "$AWS_ACCESS_KEY_ID" --profile default && aws configure set aws_secret_access_key "$AWS_SECRET_ACCESS_KEY" --profile default && aws configure set region "$AWS_DEFAULT_REGION" --profile default
+          # boto3 ignores AWS creds env vars for some reason
```

Comment:
Interesting, maybe they are executed in a different shell?

Reply:
We're running cmd.Exec and passing env, though somehow boto3 refuses to correctly see these env vars